spec/javascripts/customJestMatchers.ts

```diff
@@ -5,9 +5,9 @@ import {
 } from 'jest-matcher-utils'
 
 const toBeWithinSecondsFrom = (actual: Date, expected: Date, seconds = 1) => {
-  const diff = Math.abs(expected.getTime() - actual.getTime())
-  const pass = diff <= seconds * 1000
```

Comment:
previously it was in millis, correct? Then it makes sense unless there are also users of  the method to also change